library/src/image.cxx

```diff
@@ -175,36 +177,42 @@ image::image(unsigned int width, unsigned int height, unsigned int channelCount,
 }
 
 //----------------------------------------------------------------------------
-image::image(const std::string& path)
+image::image(const fs::path& filePath)
   : Internals(new image::internals())
 {
   detail::init::initialize();
 
-  std::string fullPath = vtksys::SystemTools::CollapseFullPath(path);
-  if (!vtksys::SystemTools::FileExists(fullPath))
+  try
   {
-    throw read_exception("Cannot open file " + path);
-  }
+    if (!fs::exists(filePath))
+    {
+      throw read_exception("Cannot open file " + filePath.string());
```

Comment:
Can we cover it?

Reply:
its supposed to be already covered, this is what I see locally at least.